sdk/tests/tutorials/instruments/test_complex_instruments.py

```diff
@@ -106,3 +106,70 @@ def test_create_fx_option(self):
         self.assertEqual(
             saved_fx_option.is_delivery_not_cash, fx_option.is_delivery_not_cash
         )
+
+    def test_create_term_deposit(self):
+
+        term_deposit = models.TermDeposit(
+            start_date=datetime(2020, 2, 5, 00, tzinfo=pytz.utc),
+            maturity_date=datetime(2020, 2, 8, 00, tzinfo=pytz.utc),
+            contract_size=1000000,
+            flow_convention=models.FlowConventions(
+                scope=None,
+                code=None,
+                currency="GBP",
+                payment_frequency="6M",
+                roll_convention="MF",
+                day_count_convention="Act365",
+                payment_calendars=[],
+                reset_calendars=[],
+                settle_days=1,
+                reset_days=0
+                ),
+            rate=0.03,
+            instrument_type="TermDeposit"
+        )
+        upsert_term_deposit = self.instruments_api.upsert_instruments(request_body={
+        "upsert_request_001": models.InstrumentDefinition(
+            name="Term Deposit Test3%",
```

Comment:
Me being nosy/pedantic (sorry): I think we should name it "Term Deposit Example". That's how I understand the Team SE approach anyway.

Similar on line 134.

As I presume this code to showcase (than actually test).

Reply:
+1, I think we generally try to avoid "demo" and "test" in our examples across the sample notebooks and SDK tests.